metricbeat/module/kubernetes/controllermanager/_meta/fields.yml

```diff
@@ -0,0 +1,128 @@
+- name: controllermanager
+  type: group
+  description: >
+    Controller manager metrics
+  release: ga
+  fields:
+    - name: handler
+      type: keyword
+      description: >
+        Request handler
+    - name: code
+      type: keyword
+      description: >
+        HTTP code
+    - name: method
+      type: keyword
+      description: >
+        HTTP method
+    - name: host
+      type: keyword
+      description: >
+        Request host
+    - name: name
+      type: keyword
+      description: >
+        Name for the resource
+    - name: zone
+      type: keyword
+      description: >
+        Infrastructure zone
+    - name: process
+      type: group
+      fields:
+        - name: cpu.sec
+          type: double
+          description: CPU seconds
+        - name: memory.resident.bytes
+          type: long
+          format: bytes
+          description: Bytes in resident memory
+        - name: memory.virtual.bytes
+          type: long
+          format: bytes
+          description: Bytes in virtual memory
+        - name: fds.open.count
+          type: long
+          description: Number of open file descriptors
+        - name: started.sec
+          type: double
+          description: Seconds since the process started
+    - name: http
+      type: group
+      fields:
+        - name: request.duration.us.percentile.*
+          type: object
+          object_type: double
+          description: Request duration microseconds percentiles
+        - name: request.duration.us.sum
+          type: double
+          description: Request duration microseconds cumulative sum
+        - name: request.duration.us.count
+          type: long
+          description: Request count for duration
+        - name: request.size.bytes.percentile.*
+          type: object
+          object_type: long
+          description: Request size percentiles
```

Comment:
These percentiles are not used?

Reply:
that's right, this is a new module, not being used yet.

Reply:
Maybe we should remove these unused fields, as they will appear in documentation and people can expect them.

Reply:
`http.request.size.bytes.precentile` is part of the metricset
you mean you want to remove it?

Reply:
Ok, keep them if they are part of the metricset, this is what I meant with being used :slightly_smiling_face: 

I guess they are generated by:
```
 			"http_request_size_bytes":                     prometheus.Metric("http.request.size.bytes"), 
```